ReleaseProcessAutomation.Tests/Steps/Releases/ReleaseRcStepTests.cs

```diff
@@ -0,0 +1,231 @@
+﻿// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under 
+// the Apache License, Version 2.0 (the "License"); you may not use this 
+// file except in compliance with the License.  You may obtain a copy of the 
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT 
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the 
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+
+using System;
+using System.IO;
+using Moq;
+using NUnit.Framework;
+using ReleaseProcessAutomation.Configuration;
+using ReleaseProcessAutomation.Configuration.Data;
+using ReleaseProcessAutomation.Extensions;
+using ReleaseProcessAutomation.Git;
+using ReleaseProcessAutomation.ReadInput;
+using ReleaseProcessAutomation.Scripting;
+using ReleaseProcessAutomation.SemanticVersioning;
+using ReleaseProcessAutomation.Steps.PipelineSteps;
+using Spectre.Console;
+
+namespace ReleaseProcessAutomation.Tests.Steps.Releases
+{
+  [TestFixture]
+  internal class ReleaseRcStepTests
+  {
+    [SetUp]
+    public void Setup ()
+    {
+      _gitClientStub = new Mock<IGitClient>();
+      _inputReaderMock = new Mock<IInputReader>();
+      _ancestorStub = new Mock<IAncestorFinder>();
+      _msBuildInvokerMock = new Mock<IMSBuildCallAndCommit>();
+      _continueAlphaBetaMock = new Mock<IContinueAlphaBetaStep>();
+
+      _consoleStub = new Mock<IAnsiConsole>();
+
+      var path = Path.Join(TestContext.CurrentContext.TestDirectory, c_configFileName);
+      _config = new ConfigReader().LoadConfig(path);
+
+    }
+
+    private Mock<IAnsiConsole> _consoleStub;
+    private Mock<IGitClient> _gitClientStub;
+    private Mock<IInputReader> _inputReaderMock;
+    private Configuration.Data.Config _config;
+    private Mock<IAncestorFinder> _ancestorStub;
+    private Mock<IMSBuildCallAndCommit> _msBuildInvokerMock;
+    private Mock<IContinueAlphaBetaStep> _continueAlphaBetaMock;
+    private const string c_configFileName = "ReleaseProcessScript.Test.Config";
+
+    [Test]
+    public void Execute_callsNextVersionFromDevelop_WithProperVersionCollection ()
```

Comment:
please check other methods on this fixtures for similar occurances of the recommended changes